exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillResultSetImpl.java

```diff
@@ -850,9 +851,9 @@ public void moveToCurrentRow() throws SQLException {
   }
 
   @Override
-  public DrillStatementImpl getStatement() {
+  public AvaticaStatement getStatement() {
```

Comment:
One reason for leaving it in (after changing the return type back) is to hold that comment pointing out that this method doesn't call checkNotClosed() as most other methods do.

Fixed copy/paste/forgot-to-edit error "close()"  to "getStatement()" in comment.